src/transport/SecureSessionMgr.h

```diff
@@ -195,6 +196,12 @@ class DLL_EXPORT SecureSessionMgr : public TransportMgrDelegate
      */
     CHIP_ERROR Init(NodeId localNodeId, System::Layer * systemLayer, TransportMgrBase * transportMgr);
 
+    /**
+     * @brief
+     *   Return the transport type used by current SecureSessionMgr.
```

Comment:
I'm not sure what "current SecureSessionMgr" means here.  We're calling a method on a `SecureSessionMgr`.  Should this say "by the current connection to the node with id peerNodeId", perhaps?  And document what happens when there is no such connection?

Reply:
Acked